src/components/Infopic/Infopic.tsx

```diff
@@ -36,32 +36,32 @@ const TextComponent = ({
   buttonUrl,
 }: Omit<InfopicProps, "sectionIndex" | "image" | "alt">) => {
   return (
-    <div className="lg:max-w-lg">
-      <div className="mt-4">
-        <div>
-          <p className="text-subtitle pb-4 uppercase tracking-widest">
-            {subtitle}
-          </p>
-        </div>
-        <h1 className="text-secondary text-5xl font-semibold pb-4">{title}</h1>
-      </div>
-
-      <section aria-labelledby="information-heading" className="mt-4">
-        <div className="mt-4 space-y-6">
+    <div className="flex flex-col gap-4 lg:px-8">
+      {subtitle && (
+        <p className="text-subtitle uppercase tracking-widest">{subtitle}</p>
+      )}
+      {title && (
+        <h1 className="text-secondary text-5xl font-semibold">{title}</h1>
+      )}
+      {title && (
+        <section aria-labelledby="information-heading">
           <p className="text-paragraph text-xl">{description}</p>
-        </div>
-      </section>
-
-      <div className="pb-4 text-lg font-semibold uppercase tracking-wid">
-        <a
-          className="inline-flex text-secondary font-semibold text-center underline uppercase pt-4"
-          href={buttonUrl}
-        >
-          {buttonLabel}
+        </section>
+      )}
+      {buttonLabel && buttonUrl && (
+        <div className="text-lg font-semibold uppercase tracking-wid">
+          <a
+            className="flex gap-2 text-secondary font-semibold underline uppercase"
+            href={buttonUrl}
+          >
```

Comment:
question(non-blocking): should this be using the LinkComponent instead? Although this might have to handle both internal and external links.

Reply:
oh, does that mean I should add Nextjs as a dependency, cus currently it seems like we're just using vanilla React